dev-support/jenkins.sh

```diff
@@ -197,7 +197,7 @@ function run_ci() {
 
   # don't let these tests cause -1s because we aren't really paying that
   # much attention to them
-  YETUS_ARGS+=("--tests-filter=checkstyle")
+  YETUS_ARGS+=("--tests-filter=checkstyle,test4tests")
```

Comment:
I opened HADOOP-19223 (https://github.com/apache/hadoop/pull/6937) to make this change independently.

Reply:
Okay, test passed, let me revert this change.